yorkie/src/main/kotlin/dev/yorkie/core/Client.kt

```diff
@@ -32,17 +32,21 @@ import io.grpc.StatusException
 import io.grpc.android.AndroidChannelBuilder
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Deferred
-import kotlinx.coroutines.Job
+import kotlinx.coroutines.ExperimentalCoroutinesApi
```

Comment:
`ExperimentalCoroutinesApi` annotation is used because `flatMapLatest` is still an experimental API.
But the code hasn't changed for over 4 years now so it should be pretty safe.
and according to [this](https://github.com/Kotlin/kotlinx.coroutines/issues/3168) it would probably go stable on next minor release.

Reply:
Thank you for the detailed description.